settings.gradle

```diff
@@ -11,3 +11,6 @@ project(':testcontainers').projectDir = 'modules/testcontainers' as File
 
 include 'model'
 project(':model').projectDir = 'modules/model' as File
+
+include 'genesis'
```

Comment:
I guess it's applicable to any block, not just genesis? maybe name everything related in more general way?